README.md

```diff
@@ -2,6 +2,9 @@
 
 Create React apps with no build configuration.
 
+* [Getting Started](#getting-started) – How to create a new app.
+* [User Guide](https://github.com/facebookincubator/create-react-app/blob/master/template/README.md) – How to develop apps bootstrapped with Create React App.
```

Comment:
Much better. Let’s also rename `How Do I...` below section into `User Guide` for consistency.